<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="f78288de-03d9-4c89-8d33-ed49306dc734" name="Admin@ADMIN-PC 2013-01-25 13:09:29" runUser="Admin-PC\Admin" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="615e9dae-ad29-4af0-995d-62fcba7df938">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment runDeploymentRoot="Admin_ADMIN-PC 2013-01-25 13_09_29">
      <DeploymentItem filename="C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\bin\Release\WindowsFormsApplication4.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific>
        <UnitTestRunConfig testTypeId="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b">
          <AssemblyResolution>
            <TestDirectory useLoadContext="true" />
          </AssemblyResolution>
        </UnitTestRunConfig>
        <WebTestRunConfiguration testTypeId="4e7599fa-5ecb-43e9-a887-cd63cf72d207">
          <Browser name="Internet Explorer 7.0">
            <Headers>
              <Header name="User-Agent" value="Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)" />
              <Header name="Accept" value="*/*" />
              <Header name="Accept-Language" value="{{$IEAcceptLanguage}}" />
              <Header name="Accept-Encoding" value="GZIP" />
            </Headers>
          </Browser>
        </WebTestRunConfiguration>
      </TestTypeSpecific>
      <AgentRule name="LocalMachineDefaultRole">
        <DataCollectors>
          <DataCollector uri="datacollector://microsoft/CodeCoverage/1.0" assemblyQualifiedName="Microsoft.VisualStudio.TestTools.CodeCoverage.CoveragePlugIn, Microsoft.VisualStudio.QualityTools.Plugins.CodeCoverage, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" friendlyName="Code Coverage">
            <Configuration>
              <CodeCoverage xmlns="">
                <Regular>
                  <CodeCoverageItem binaryFile="C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\StegAppTest\bin\Release\StegAppTest.dll" pdbFile="C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\StegAppTest\bin\Release\StegAppTest.instr.pdb" instrumentInPlace="true" />
                  <CodeCoverageItem binaryFile="C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\bin\Release\WindowsFormsApplication4.exe" pdbFile="C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\bin\Release\WindowsFormsApplication4.instr.pdb" instrumentInPlace="true" />
                </Regular>
              </CodeCoverage>
            </Configuration>
          </DataCollector>
        </DataCollectors>
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2013-01-25T13:09:29.2128000+00:00" queuing="2013-01-25T13:10:05.9928000+00:00" start="2013-01-25T13:10:06.5638000+00:00" finish="2013-01-25T13:11:21.8698000+00:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <ResultFiles>
      <ResultFile path="ADMIN-PC\data.coverage" />
    </ResultFiles>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="RegisterFnTest1" storage="c:\users\admin\documents\visual studio 2010\projects\stegapp\stegapp\stegapptest\bin\release\stegapptest.dll" id="aef830a2-ecaa-4b24-01f8-0ad0a7d91840">
      <Execution id="30d18fe7-fdef-49d3-881f-c5155989a9df" />
      <TestMethod codeBase="C:/Users/Admin/Documents/Visual Studio 2010/Projects/StegApp/StegApp/StegAppTest/bin/Release/StegAppTest.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="StegAppTest.RegisterTest, StegAppTest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="RegisterFnTest1" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="aef830a2-ecaa-4b24-01f8-0ad0a7d91840" executionId="30d18fe7-fdef-49d3-881f-c5155989a9df" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="30d18fe7-fdef-49d3-881f-c5155989a9df" testId="aef830a2-ecaa-4b24-01f8-0ad0a7d91840" testName="RegisterFnTest1" computerName="ADMIN-PC" duration="00:00:48.4087546" startTime="2013-01-25T13:10:28.9248000+00:00" endTime="2013-01-25T13:11:20.4258000+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="30d18fe7-fdef-49d3-881f-c5155989a9df">
      <Output>
        <ErrorInfo>
          <Message>Test method StegAppTest.RegisterTest.RegisterFnTest1 threw exception: 
System.Data.EntityException: The underlying provider failed on Open. ---&gt; System.Data.SqlClient.SqlException: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)</Message>
          <StackTrace>    at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning()
   at System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity)
   at System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, SqlConnection owningObject)
   at System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, Boolean redirectedUserInstance, SqlConnection owningObject, SqlConnectionString connectionOptions, TimeoutTimer timeout)
   at System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(SqlConnection owningObject, TimeoutTimer timeout, SqlConnectionString connectionOptions, String newPassword, Boolean redirectedUserInstance)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, Object providerInfo, String newPassword, SqlConnection owningObject, Boolean redirectedUserInstance)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnection owningConnection, DbConnectionPool pool, DbConnectionOptions options)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionPool.GetConnection(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionFactory.GetConnection(DbConnection owningConnection)
   at System.Data.ProviderBase.DbConnectionClosed.OpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory)
   at System.Data.SqlClient.SqlConnection.Open()
   at System.Data.EntityClient.EntityConnection.OpenStoreConnectionIf(Boolean openCondition, DbConnection storeConnectionToOpen, DbConnection originalConnection, String exceptionCode, String attemptedOperation, Boolean&amp; closeStoreConnectionOnFailure)
 --- End of inner exception stack trace ---
    at System.Data.EntityClient.EntityConnection.OpenStoreConnectionIf(Boolean openCondition, DbConnection storeConnectionToOpen, DbConnection originalConnection, String exceptionCode, String attemptedOperation, Boolean&amp; closeStoreConnectionOnFailure)
   at System.Data.EntityClient.EntityConnection.Open()
   at System.Data.Objects.ObjectContext.EnsureConnection()
   at System.Data.Objects.ObjectContext.SaveChanges(SaveOptions options)
   at WindowsFormsApplication4.DataAccess.DataAccess.InsertUser(String username, String password, String firstname, String lastname, String Town, String County, String Country) in C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\DataAccess\DataAccess.cs:line 133
   at WindowsFormsApplication4.Register.RegisterFn(String username, String password, String firstname, String surname, String town, String county, String country) in C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\Register.cs:line 33
   at StegAppTest.RegisterTest.RegisterFnTest1() in C:\Users\Admin\Documents\Visual Studio 2010\Projects\StegApp\StegApp\StegAppTest\RegisterTest.cs:line 150
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>